### DISCUSSION TITLE
[@types/mixpanel] no identify function??

### DISCUSSION THREAD

**Discussion by soldey** [DefinitelyTyped/DefinitelyTyped#56563] [Issues with a @types package]
"@types/mixpanel": "^2.14.3",
    "mixpanel": "^0.13.0",
  Why is there no identify function?

[a comment by an automated account is not shown]

**Comment by danmana** [comment]
Looks like it might be missing on purpose - see  [Where is mixpanel.identify](https://www.npmjs.com/package/mixpanel#:~:text=Where%20is%20mixpanel.identify) in the FAQ and https://github.com/mixpanel/mixpanel-node/issues/13
